src/guide/troubleshooting/configuration-issues.md

```diff
@@ -8,6 +8,15 @@ common source of issues in Iroha.
 If the issue you are experiencing is not described here, contact us via
 [Telegram](https://t.me/hyperledgeriroha).
 
-<!-- TODO: add -->
+## Fixing a `genesis` file issue on a Docker-compose setup
 
-**TBD**
+If you are using the Docker-compose version of Iroha and you have pulled the Iroha code the first time long ago, you may be surprised with Iroha working incorrectly. A mismatch between the versions of Iroha may lead to one of the peer containers failing with an error, related to the genesis file, i.e. a file that is needed to initialize the Iroha peer.
+
+To find out if this situation is applicable, you need to start by check the current containers using the `docker ps` command.
+Depending on the version, you may see either `hyperledger/iroha2:dev` or `hyperledger/iroha2:lts` containers. Pay attention to the number of the Iroha peer containers you are seeing in the `docker ps` output. By default, there are 4 peers configured in `docker-compose.yml` for Iroha, although you may have changed that value. Now, you may see that the first container that should have been running Iroha just exited with an error, while three other containers remain active.
+
+You can get a better understanding of this situation by checking the logs. If you've started your Iroha in daemon mode with `docker compose up -d`, you should refer to the `docker compose logs` command. You may also be running it directly with the `docker compose up` command. So, to check if you have this issue, look for an error text, containing "Failed to deserialize raw genesis block".
```

Comment:
```suggestion
When you are using the Docker-compose version of Iroha, you might encounter the issue of one of the peer containers failing with the `Failed to deserialize raw genesis block` error. This happens if there is a mismatch between Iroha versions, meaning an Iroha peer cannot be initialized with the given genesis file.

If one of the peers is failing and it's been a while since you pulled the Iroha code for the first time, it's safe to assume the outdated genesis file is the cause. Here is how you can make sure Iroha is working incorrectly for exactly this reason:

1. Use `docker ps` to check the current containers. Depending on the version, you will see either `hyperledger/iroha2:dev` or `hyperledger/iroha2:lts` containers. Check the number of Iroha peer containers in the `docker ps` output. By default, there are 4 peers configured in `docker-compose.yml` for Iroha, although you may have changed that value. You will see that the first container that should have been running Iroha just exited with an error, while three other containers remain active.

2. Check the logs and look for the `Failed to deserialize raw genesis block` error. If you started your Iroha in daemon mode with `docker compose up -d`, use `docker compose logs` command.
```